SzczepionkaAPI/src/main/java/com/szczepionka/service/AppointmentService.java

```diff
@@ -33,9 +35,9 @@ public Appointment newAppointment(PatientDTO patientDTO, long locationId) {
         Patient patient = patientService.addPatient(patientDTO);
 
         Appointment appointment = Appointment.builder()
-                .id(patient.getId())
-                .appointmentDate(createAppointmentDate())
-                .appointmentTime(createAppointmentTime())
+                .id(patient.getId()) //FIXME patientID always 0
```

Comment:
Instead of this comment, create please task on Trello to not forget about this bug.

Reply:
I have marked is as a comment and as a task on trello, from my experience its easier to track errors like that in the code so you can see directly where is the problem mentioned in trello. Also it is easier to find it, when you want to come back to it later, as those are listed in intellij